src/main/java/org/broadinstitute/hellbender/tools/exome/acsconversion/ACSModeledSegmentUtils.java

```diff
@@ -0,0 +1,139 @@
+package org.broadinstitute.hellbender.tools.exome.acsconversion;
+
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.ACNVModeledSegment;
+import org.broadinstitute.hellbender.tools.exome.Genome;
+import org.broadinstitute.hellbender.tools.exome.ModeledSegment;
+import org.broadinstitute.hellbender.tools.exome.SegmentUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableUtils;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Static class for writing out legacy AllelicCapSeg formatted files from ACNV output.
+ */
+public class ACSModeledSegmentUtils {
+    /**
+     * Writes a list of segments represented by {@link ACNVModeledSegment} to a file with header:
+     * <p>
+     *      Chromosome | Start.bp | End.bp | n_probes | length | n_hets | f | tau | sigma.tau | mu.minor | sigma.minor | mu.major | sigma.major | SegLabelCNLOH
+     * </p>
+     *
+     * This header is produced by the Broad Institute CGA tool, AllelicCapSeg.  Please note that SegLabelCNLOH is not populated properly.
+     */
+    public static void writeACNVModeledSegmentFileAsAllelicCapSegFile(final File outFile,
+                                                                      final List<ACNVModeledSegment> acnvModeledSegments,
```

Comment:
If writeSegmentFile in SegmentUtils was made public, this code could be reduced to a one-liner (as in the methods calling the public readSegmentFile method above).